mqtt 3/1.0.0/modules/ROOT/pages/mqtt3-connector-studio-configuration.adoc

```diff
@@ -0,0 +1,412 @@
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `mqtt 3` in the search field.
+. Click *MQTT 3 Connector* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-input-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these input sources to use with JMS Connector:
+
+* *MQTT3 > On New Message* +
+Initiates a flow by listening for incoming messages
+* *HTTP > Listener* +
+Initiates a flow each time it receives a request on the configured host and port
+* *Scheduler* +
+Initiates a flow when a time-based condition is met
+
+For example, to configure an *On New Message* source, follow these steps:
+
+. In the *Mule Palette* view, select *MQTT3 > On New Message*.
+. Drag *On New Message* to the Studio canvas.
+. In the *On New Message* configuration screen, optionally change the value of the *Display Name* field.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the source in the app.
+. In the *MQTT3 Config* window, for *Connection*, select either one of the connection types to provide to this configuration:
+
+* *MQTT3 URL Connection*
+* *MQTT3 Fail-Over Connection*
+* *MQTT3 Form Connection*
+
+[start=6]
+. Set the mandatory *Client ID* to identify an MQTT connection to a broker.
+. On the *General* tab also specify the connection information for the connector, such as required libraries for the broker and advanced connection options.
+. On the *LWT* tab, optionally specify last will and testament message.
+. On the *TLS/SSL* tab, optionally specify a TLS configuration.
+. On the *Advanced* tab, optionally specify a reconnection strategy.
+. Click *OK* to close the window.
+. In the *On New Message* configuration screen, in *Topics*, select *Edit inline* to list the topics that the listener subscribes to.
+. Click the plus sign (*+*) to configure a topic.
+. In the *Topic* configuration screen, set *Topic filter* that represents a single or multilevel subscription to a topic.
+. Set *QoS* to any of the following quality of service levels:
++
+* *AT_MOST_ONCE*
+* *AT_LEAST_ONCE*
+* *EXACTLY_ONCE*
+. Click *Finish*.
+
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for MQTT 3 Connector, follow these steps:
+
+. In the *Mule Palette* view, select *MQTT3* and then select the desired operation.
+. Drag the operation onto the Studio canvas and to the right of the input source.
+
+.MQTT 3 Connector operations
```

Comment:
We don't need to add captions for images (creates redundancy for screen readers). Only alt text is required.